README.md

```diff
@@ -5,23 +5,181 @@ Record any website using our recorder, or deploy our service to manage it for yo
 
 ## How it works
 
-The recorder launches Chrome and navigates to the supplied url, grabs audio from pulse and video from a virtual frame buffer, and feeds them into GStreamer.
-You can write the output as mp4 to a file or upload it to s3, or forward the output to one or multiple rtmp streams.
+The recorder launches Chrome and navigates to the supplied url, grabs audio from pulse and video from a virtual 
+frame buffer, and feeds them into GStreamer. You can write the output as mp4 to a file or upload it to s3, or forward 
+the output to one or multiple rtmp streams.
+
+## Quick start
+
+Start by filling in a `config.yaml`:
+```
+api_key: <livekit-server-api-key>
+api_secret: <livekit-server-api-secret>
+ws_url: <livekit-server-ws-url>
+file_output:
+    local: true
+```
+
+Next, create a `request.json`:
+```json
+{
+  "url": "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
+  "filepath": "/out/demo.mp4"
+}
+```
+
+Start the recording:
+```shell
+mkdir -p ~/livekit/recordings
+
+docker run --rm --name demo \
+    -e LIVEKIT_RECORDER_CONFIG="$(cat config.yaml)" \
+    -e RECORDING_REQUEST="$(cat basic.json)" \
+    -v ~/livekit/recordings:/out \
+    livekit/livekit-recorder
+```
+
+Then, to stop the recording:
+```shell
+docker stop demo
+```
+
+You should find a `~/livekit/recordings/demo.mp4`.
+
+## Uploading to S3
+
+Update your config.yaml and replace
+```yaml
+file_output:
+    local: true
+```
+with
+```yaml
+file_output:
+    s3:
+        access_key: <s3-access-key>
+        secret: <s3-secret>
+        region: <s3-region>
+        bucket: <s3-bucket>
+```
+
+Create a `s3.json`:
+```json
+{
+    "url": "https://www.youtube.com/watch?v=BHACKCNDMW8",
```

Comment:
I'm not sure if we should keep pointing people to youtube. I think majority of users are coming here to figure out how to record a room. keep mentioning YT may confuse users what this is actually used for.